packages/libs/react-ui/CONTRIBUTING.md

```diff
@@ -0,0 +1,208 @@
+# Contributing to @kadena/react-ui
+
+This is an aggregation of the conventions that we have picked up while
+developing this component library. This library is still in its early stages
+development so please make any suggestions if you see ways it can be improved!
```

Comment:
```suggestion
developing this component library. This library is still in its early stages
of development so please make any suggestions if you see ways it can be improved!
```